kotlin.web.demo.frontend/src/views/tabs/OutputView.kt

```diff
@@ -69,42 +69,70 @@ class OutputView(val element: HTMLElement) {
 
     fun printException(exception: dynamic) {
         printErrorLine()
-        printErrorLine(
-                if (exception.message != null) {
-                    "Exception in thread \"main\" " + exception.fullName + ": " + unEscapeString(exception.message)
-                } else {
-                    "Exception in thread \"main\" " + exception.fullName
-                }
-        )
-        printExceptionBody(exception)
+        var currentException = exception
+        var isSecurityException = false
+        val accessControlException = "java.security.AccessControlException"
+        while (currentException != null) {
```

Comment:
I think is better to add method to extract security exception or null if this is not the case and another method that will print security exception.